README.md

```diff
@@ -1,51 +1,66 @@
-# loopback4-extension-grpc
+# gRPC Extension for LoopBack 4
 
 [![Join the chat at https://gitter.im/strongloop/loopback4-extension-grpc](https://badges.gitter.im/strongloop/loopback4-extension-grpc.svg)](https://gitter.im/strongloop/loopback4-extension-grpc?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge)
 
-gRPC Extension for LoopBack 4
-
-## Summary
-A gRPC extension for LoopBack Next.
-
 ## Overview
-TODO.
+The `@loopback/grpc` component enables LoopBack 4 as a Grpc Server. Also it provides with a @grpc decorator to define your RPC Method implementations from your Application Controllers.
 
 ## Installation
-Get started by either downloading this project or cloning it as follows:
+Install the `@loopback/grpc` component in your LoopBack 4 Application.
 
 ```sh
-$ git clone https://github.com/strongloop/loopback4-extension-grpc.git
-$ cd loopback4-extension-grpc && npm install
+$ npm install --save @loopback/grpc
 ```
 
-## Basic Usage
-TODO 
-
-### Components
-TODO.
-
-### Controllers
-TODO.
+## Component Configuration
+```js
+import {Application} from '@loopback/core';
+import {GrpcComponent, GrpcConfig} from '@loopback/grpc';
+import {MyGreeter} from './MyGreeter';
 
-### Decorators
-TODO.
+const config: GrpcConfig = {
+    proto: './file.proto',
+    package: 'myawesomepackage'
+}
 
-### Mixins
-TODO. 
+const app = new Application({
+    components: [GrpcComponent],
+    grpc: config
+});
 
-### Providers
-TODO. 
+app.controller(MyGreeter});
 
-### Repositories
-TODO.
+await app.start();
+```
 
-### Project Layout
+## Grpc Decorator
+The `@loopback/grpc` component provides you with a handy decorator to implement GRPC Methods.
+
+```js
+import {grpc} from '@loopback/grpc';
+// You create the following types according your own proto.file
+import {Greeter, HelloRequest, HelloReply} from './types';
+/**
+ * @class MyGreeter
+ * @description Implements grpc proto service
+ **/
+export class MyGreeter implements Greeter {
+    // Tell LoopBack that this is a Service RPC implementation
+    @grpc()
+    SayHello(request: HelloRequest): HelloReply {
```

Comment:
The method name is typically camelCase.

Reply:
Do we want to support separate parameters, such as `sayHello(prefix: string, name: string)`? The HelloRequest will have `prefix` and `name` properties. Maybe ES6 parameter destructuring is good enough.

Reply:
Yeah you are right I took it from the grpc examples but these are language agnostic, in this case the convention is camelCase so I'll change that. 👍 

About the parameters I would rather not. You know, I like the idea of passing an entire dependency instead of lots of parameters.

Do you see any benefit on passing each of the parameters instead of the entire object?

It could be a configurable option though, I would like the entire object to be default, but an option to pass the args as params should be easy to implement anyway.